core/common/src/main/java/org/phenotips/remote/common/internal/api/DefaultPatientToJSONConverter.java

```diff
@@ -112,6 +108,30 @@ public JSONObject convert(Patient patient, boolean removePrivateData, int includ
         return json;
     }
 
+    private static JSONObject contact(Patient patient) {
+        // Default contact info
+        String name = "PhenomeCentral";
+        String institution = "PhenomeCentral";
+        String href = "mailto:matchmaker@phenomecentral.org";
+
+        PatientData<String> data = patient.getData("contact");
+        if (data != null && data.isNamed()) {
+            name = data.get("name");
+            institution = data.get("institution");
```

Comment:
Do we always have institution defined in PT/PC? Or should we also check for isBlank() and only replace default (PhenomeCentral) if not blank? Or blank is actually expected (as in "no institution")?